sw/device/sca/ecc_serial.c

```diff
@@ -202,15 +249,6 @@ static void p256_ecdsa_sign(const uint32_t *msg, const uint32_t *private_key_d,
  */
 static void ecc256_ecdsa(const uint8_t *ecc256_secret_k_bytes,
```

Comment:
This function has a description in the comment above that also needs to be aligned.

Reply:
In addition, both the `ecc256_secret_k_bytes` and `secret_k_len` arguments are no longer used. So they should probably be removed from the function declaration/header.

Reply:
The first point here (aligning function documentation) is still open. Can you please address this, too?

Reply:
Ah, good catch, thanks! I updated the function documentation too.